core/src/main/kotlin/com/malinskiy/marathon/io/CachedFileHasher.kt

```diff
@@ -2,18 +2,19 @@ package com.malinskiy.marathon.io
 
 import org.apache.commons.collections4.map.LRUMap
 import java.io.File
+import java.util.*
 
 class CachedFileHasher(
     private val delegate: FileHasher,
     cacheCapacity: Int = DEFAULT_CACHE_CAPACITY
 ) : FileHasher {
 
-    private val cache = LRUMap<File, String>(cacheCapacity)
+    private val cache = Collections.synchronizedMap(LRUMap<File, String>(cacheCapacity))
 
     override suspend fun getHash(file: File): String =
         cache.getOrPut(file) { delegate.getHash(file) }
 
     private companion object {
-        private const val DEFAULT_CACHE_CAPACITY = 128
+        private const val DEFAULT_CACHE_CAPACITY = 512
```

Comment:
What's the reason to increase the capacity?
Is it implicitly related to synchronisation?

Reply:
no, it's not related. just increased as the cache is cheap and we have ~400 modules so it's better just to keep everything in the cache